CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 Contributing to ITK
 ===================
 
+Welcome to the ITK! We're excited you're here and want to contribute.
```

Comment:
I'd reword the first part either as

`Welcome to ITK !`

or

`Welcome to the Insight Toolkit ! `

Reply:
OK. looks like 131c019 addresses this. Thanks @thewtex .

Reply:
@jhlegarreta thanks for the review and suggestion -- updated to `Welcome to the Insight Toolkit (ITK) !`